controllers/adminnetworkpolicy_controller.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
For reviewers: We shouldn't need anything in `controllers/` for now since the implementation of the ANP API will occur in CNIs so either I can remove it now or in a followup PR